modules/tensor_mechanics/test/tests/ad_finite_elastic/tests

```diff
@@ -0,0 +1,26 @@
+[Tests]
+  [./finite_elastic-noad]
+    type = 'Exodiff'
+    input = 'finite_elastic.i'
+    exodiff = 'finite_elastic_out.e'
+    requirement = 'We shall be able to reproduce the results of the hand-coded simulation using automatic differentiation in the production stress divergence kernel'
+    allow_test_objects = true
```

Comment:
Do we need this anymore? Also, we need `design` and `issues`

Reply:
design and issues are up top

Reply:
Ugh, missed some of the `allow_test_objects`. Have it in my next PR though (bf414204eba0b2f92fe77a76bc1825b6e7eab5d8). Is that cool?

Reply:
Yea, that's fine. Or if @bwspenc requests changes here